bindings/python/test_m68k.py

```diff
@@ -40,12 +40,14 @@
 	16: "Absolute Data Addressing  - Short",
 	17: "Absolute Data Addressing  - Long",
 	18: "Immidate value",
-}    
+}
 
 def print_insn_detail(insn):
     if len(insn.operands) > 0:
         print("\top_count: %u" % (len(insn.operands)))
 
+	print("\tgroups_count: %u" % len(insn.groups))
```

Comment:
Thanks for the PR. Would it be fine to submit it directly to the "next" branch directly or should i merge it within this PR?